packages/bits/src/lib/image/image.component.ts

```diff
@@ -66,6 +74,8 @@ export class ImageComponent implements OnInit, AfterViewInit {
      */
     @Input() public autoFill: boolean;
 
+    public ariaLabel: string;
```

Comment:
The `ariaLabel` need to be an input. It gives the ability for the user to set their own `ariaLabel`.